src/main/scala/com/gu/riffraff/artifact/RiffRaffArtifact.scala

```diff
@@ -45,10 +45,10 @@ object RiffRaffArtifact extends AutoPlugin {
     lazy val riffRaffUploadArtifactBucket = settingKey[Option[String]]("Bucket to upload artifacts to")
     lazy val riffRaffUploadManifestBucket = settingKey[Option[String]]("Bucket to upload manifest to")
 
-    lazy val riffRaffNotifyTeamcity = settingKey[Boolean]("Report artifacts to Teamcity if true")
+    lazy val riffRaffNotify = taskKey[Unit]("Task to notify teamcity")
```

Comment:
Ooh, good idea. That makes sense to me. I'd give it a slightly bigger version bump and email dig.dev, but that seems a better solution.